src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -1493,7 +1584,7 @@ public void receiveObject(Serializable object, ConnectionHandlerID connectionHan
 
                             default:
                                 /**
-                                 * Non-matching contest Ids - do not process packet
+                                 * Non-matching contest Ids - do not process packet <-- Huh?  It looks like the code below DOES "process the packet"?  jlc
```

Comment:
No change requested.
In the future if there is a question about the code or a comment put a TODO or SOMEDAY in front of the comment.

Reply:
Noted.
You mean like this line in InternalController:
`        // XXX this if does not make sense, should it be if serverModule? (huh? dal)`
? ;)